packages/router/src/components/Link.tsx

```diff
@@ -1,49 +1,61 @@
-import * as React from 'react'
-import type { AnchorHTMLAttributes } from 'react'
+import type * as React from 'react'
 import { useInView } from 'react-intersection-observer'
 
 import { useRouter } from '../hooks/useRouter'
 import useRoute from '../hooks/useRoute'
 
-interface TuonoLinkProps {
+interface TuonoLinkProps
+  extends Omit<React.AnchorHTMLAttributes<HTMLAnchorElement>, 'preload'> {
```

Comment:
Does `React.AnchorHTMLAttributes<HTMLAnchorElement>` have the `preload` property?

Reply:
No, I don't know how got in my mind 😩